mobile/views/components/forum_new_topic.html

```diff
@@ -0,0 +1,55 @@
+{form name="topic_form"}
+    <div class="owm_forum_topic owm_forum_topic_reply">
+        <div class="owm_forum_topic_nav">
+            <div class="owm_forum_topic_title owm_bg_color_1">
+                {input name="title"}
+            </div>
+            <div class="owm_forum_name_select owm_bg_color_1">
+                <div class="owm_select_wrap">
+                    <div class="ow_inline owm_inline">
+                        {input name="group"}
+                    </div>
+                </div>
+            </div>
+            <div class="owm_forum_topic_subscribe owm_bg_color_1">
+                <div class="owm_checkbox_wrap">
+                    <ul class="ow_checkbox_group clearfix">
+                        <li>
+                            {input name="subscribe"}
```

Comment:
you have to add error field

Example:
<div class="owm_form_label owm_small_margin_bottom">{label name='accountType'}</div>
<div class="owm_field_wrap owm_select_wrap">{input name='accountType'}</div>
<div class="owm_error_txt">{error name='accountType'}</div>